### PR TITLE
added landing page

### PR DESCRIPTION
**Pull request by omkarpatke**
done with landing page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    margin-right: 8rem;
+    margin-top: 3rem;
+}
+
+.cycle-desc{
+    font-size: 1.8rem;
+}
+
+.cycle-price{
+    margin: 10px 0 1rem 0;
```
Comment (varun1233) [inline style.css:176]: use rem

File: style.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+.footer-links-container ul li a{
+    text-decoration: none;
+    color: var(--white-color);
+    list-style: none;
+}
+
+.social-media-links{
+    display: flex;
+    gap: 10px;
```
Comment (varun1233) [inline style.css:219]: use rem

**Comment by varun1233** [comment]
image
Get Started button is breaking
Trying removing unwanted space from CSS and html

File: style.css (hunk 1 above)
Comment (omkarpatke) [reply style.css:176]: but this is less than 16px thats why I'm using px.

Comment (omkarpatke) [reply style.css:176]: but this is less than 16px thats why I'm using px.

**Review by Naman-Saxena1 — APPROVED** [review]
Looks good to me.
It is not mobile responsive but that's ok, it is not mandatory.

**Pull request merged**